packages/tauri/src-tauri/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "Padloc"
-version = "0.1.0"
+version = "4.0.0"
```

Comment:
I don't think we need to bother updating this field, since the version from `tauri.conf.json` take precedence over this one.

Reply:
I have found that to _not_ be the case for linux, for some reason, only for macos/windows. Similarly, the file name is `padloc` in Linux, but `Padloc` in macos/windows, which is strange, but not a deal-breaker, since the app is still `Padloc` everywhere.